src/main/java/com/xeeva/catalog/CostCenterPage.java

```diff
@@ -1,532 +1,583 @@
+	@FindBy(xpath=".//*[@class='gridtextLink']") private List<WebElement> lstItemNoGrid;
+	@FindBy(css=".costCentername") private Label lblCC;
+	@FindBy(xpath=".//*[@id='customfa']/tbody/tr/td/input[@class='QtyNumericTextBoxClass']") private List<WebElement> lstQty;
+	@FindBy(xpath=".//*[@title='Save changes']") private List<WebElement> lstSave;
+	@FindBy(xpath="//div[@id='divAppInfoMsg'][@class='addMessage']") private Label conformationMsg;
+	@FindBy(xpath=".//*[@value='Save Cart'][contains(@onclick, 'javascript')]") private WebElement btnSaveCart;
+
+	@FindBy(xpath="//td[2]/a[1]/input") private Button btnSaveCartCC;
+	@FindBy(xpath="//tr[2]/td[2]/a[2]/input") private Button btnShopMoreItemsCC;
+	@FindBy(xpath="//*[@id='btnCheckOut']/input") private Button btnContinueCheckoutCC;
+	@FindBy(xpath=".//*[@id='customfa']/tbody/tr/td/select") private List<Listbox> ddCCLineLevel;
+	@FindBy(xpath="//*[@id='tblbasicTable']/tbody/tr/td/div/a") private List<WebElement> lstSelectCC;
+	@FindBy(xpath="//*[@id='tblbasicTable']/tbody/tr/td/div/span") private List<WebElement> lstSelectCCValue;
+
+
+
+	@FindBy(xpath="//table[@id='customfa']/tbody/tr/td/table/tbody/tr/td/input") private List<WebElement> dateCCInputLineLevel;
+	@FindBy(xpath="//table[@id='customfa']/tbody/tr/td/table/tbody/tr/td/i") private List<WebElement> dateCCLineLevel;
+	@FindBy(xpath="//div[@id='divAPRPARList']/ul/li[4]/i") private Link lnkDateCC_HeaderLvel;
+
+	@FindBy(xpath="//div[@id='divAppInfoMsg'][@class='addMessage']") private Label lblCartItemAddedMessage;
+	@FindBy(xpath="//input[@id='txtRequiredby']") private Textbox txtReqByHeaderLevel;
+	@FindBy(xpath=".//*[@class='buttonClass'][@value='SHOP FOR MORE ITEMS']") private Button btnShopForMoreItems;
+	@FindBy(xpath=".//*[@id='customfa']/tbody/tr/td[10]/a[2]") private List<WebElement> lstEditItemGrid;
+	@FindBy(xpath=".//*[@value='CONTINUE Checkout']") private Button btnContinueCheckOut;
+	@FindBy(xpath=".//*[@id='ddlshippingaddress']") private WebElement lstShippingAdd;
+
+
+	/**Constructor**/
+	public CostCenterPage(OrasiDriver driver){
+		this.driver = driver;
+		ElementFactory.initElements(driver, this);
+	}
+
+
+	public void pageLoaded(){
+		costCenterContainer.syncVisible(20, false);
+	}
+
+	/**Page Interactions**/
+
+
+	/**
+	 * @summary Clicking Internal Comment Link
+	 * @author  Lalitha Banda
+	 * @date    15/09/16
+	 */
+	public void click_InternalComment(){
+		pageLoaded();
+		driver.executeJavaScript("arguments[0].click();", lstInternalCommentGrid.get(0));
+		//lstInternalCommentGrid.get(0).click();
+	}
+
+	/**
+	 * @summary Clicking CopyItem Link
+	 * @author  Lalitha Banda
+	 * @date    15/09/16
+	 */
+	public void click_CopyItemLink(){
+		Sleeper.sleep(5000);
+		lstCopyItemGrid.get(0).click();
+	}
+
+
+	/**
+	 * @summary Adding Internal Comment
+	 * @author  Lalitha Banda
+	 * @date    15/09/16
+	 */
+	public void AddInternalComment(String InternalComment){
+		lblCC.syncVisible(30, false);
+		lblCC.isDisplayed();
```

Comment:
If you are not doing anything with the return value of isDisplayed(), then please remove this line.